runtime/service.go

```diff
@@ -1281,7 +1281,7 @@ func (s *service) Delete(requestCtx context.Context, req *taskAPI.DeleteRequest)
 	var result *multierror.Error
 	// Trying to release stub drive for further reuse
 	err = s.containerStubHandler.Release(requestCtx, req.ID, s.driveMountClient, s.machine)
-	if err != nil {
+	if err != nil && !errors.Is(err, ErrDriveNotFound) {
 		result = multierror.Append(result, errors.Wrapf(err, "failed to release stub drive for container: %s", req.ID))
```

Comment:
Hmm, why does it release drives that are not there?

Reply:
And Delete() releases all of them regardless of they are managed or not?

Is it possible to handle that inside the implementation of Release()? (cc @Kern--)

Reply:
The problem is on task creation we know the container is mounted inside the VM by inspecting the `rootfs` on the request. On deletion however, all we have is the request ID. Another solution is to implement a stub handler function to check if the drive is in use before attempting to release, but that feels more weak than a no-op.

Reply:
I think we probably want to do the book keeping in the service to know whether the container is drive-backed or inside the VM. It's pretty easy for implicit fallback to either mask bugs later, or make debugging more difficult, or just make it harder to understand the code from an outside perspective.

Reply:
Okay, I'll try my hand at that solution and get another revision out.